python/ray/serve/config.py

```diff
@@ -258,15 +258,17 @@ def grpc_servicer_func_callable(self) -> List[Callable]:
                 if callable(imported_func):
                     callables.append(imported_func)
                 else:
-                    logger.warning(
+                    logger.exception(
```

Comment:
https://docs.python.org/3/library/logging.html#logging.Logger.exception
change to `logger.error`. 

logger.exception is used for debugging since it can include the traceback inside the exception handler.

Reply:
made the change to just raise the exception with the message, so no longer logging them separately :)